ui/Modal/Search.svelte

```diff
@@ -0,0 +1,171 @@
+<script lang="typescript">
+  import { push } from "svelte-spa-router";
+  import { createEventDispatcher } from "svelte";
+
+  import * as notification from "../src/notification";
+  import * as path from "../src/path";
+  import type { Project } from "../src/project";
+  import * as remote from "../src/remote";
+  import {
+    projectRequest as request,
+    projectSearch as store,
+    reset,
+    requestProject,
+    searchProject,
+    urnValidationStore,
+  } from "../src/search";
+  import { ValidationStatus } from "../src/validation";
+
+  import { Icon, Input } from "../DesignSystem/Primitive";
+  import { Remote, TrackToggle } from "../DesignSystem/Component";
+
+  let id: string;
+  let value: string;
```

Comment:
could you document what these are?

Reply:
Could you expand on what kind of documentation you expect when stumbling upon these? Also would it help to rename them to not require documentation?

Reply:
Yeah, renaming would probably be best. Expanding, I open this file and have no idea what `id` and `value` stand or are used for.

Reply:
Looking at the code I'd probably call this `rawUrn` or `unvalidatedUrn`.

Also is the `id` above actually the `urn`? I find the name `id` hard to discern whether it's a PeerId or a RadUrn. WHERE ARE MY TYPES AS DOCUMENTATION /_raises fist to sky_

Reply:
> Looking at the code I'd probably call this `rawUrn` or `unvalidatedUrn`.
It's validated.


It's the `id` part of of a RadUrn.